docs/source/instrument/instrument_template.md

```diff
@@ -3,5 +3,5 @@
 TODO: gather all that documentation here
```

Comment:
Is this file necessary now?  Could be removed (any links?) with consequence.

Reply:
git grep did not find any reference to it, removed.